### PR TITLE
PASE-1724: Modify the memcache dropin to use main object-cache file.

### PR DESCRIPTION
**Pull request by rutviksavsani**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -246,6 +246,7 @@ runs:
         WP_CONFIG_PATH="${WP_TESTS_DIR}/wp-tests-config.php"
 
         echo "define( 'IS_UNIT_TEST', true );" >> "${WP_CONFIG_PATH}"
+        echo "define( 'VIP_GO_APP_ENVIRONMENT', 'staging' );" >> "${WP_CONFIG_PATH}"
```
Comment (rutviksavsani) [inline action.yml:249]: Need to define the constant as [object-cache.php](https://github.com/Automattic/vip-go-mu-plugins-built/blob/5cc989e8c388f64a145fda4841f53d84b0182a5e/drop-ins/object-cache.php) needs it defined.

File: scripts/set-up-wordpress-dependencies.sh [hunk 2]
```diff
@@ -27,8 +27,12 @@ mv "${RUNNER_TEMP}/plugin-loader.php" "${WP_CONTENT_TARGET_DIR}/client-mu-plugin
 git_checkout "${WP_CONTENT_TARGET_DIR}/mu-plugins" https://github.com/Automattic/vip-go-mu-plugins-built.git 1
 
 # Install memcached drop-in.
-if [[ ! -f "${WP_CONTENT_TARGET_DIR}/object-cache.php" && -f "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache/object-cache-stable.php" ]]; then
-  ln -s "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache/object-cache-stable.php" "${WP_CONTENT_TARGET_DIR}/object-cache.php"
+if [[ ! -f "${WP_CONTENT_TARGET_DIR}/object-cache.php" && -f "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache.php" ]]; then
+  ln -s "${WP_CONTENT_TARGET_DIR}/mu-plugins/drop-ins/object-cache.php" "${WP_CONTENT_TARGET_DIR}/object-cache.php"
+else
```
Comment (rutviksavsani) [inline scripts/set-up-wordpress-dependencies.sh:32]: Adding an else condition as the object-cache.php file already exists and hence link was not created. In that case we delete the file and create a new one.

File: action.yml (hunk 1 above)
Comment (ethitter) [inline action.yml:249]: Why is this needed? I don't see any check of the              `VIP_USE_NEXT_OBJECT_CACHE_DROPIN` constant in the new loader, just a check for the PHP extension

Comment (rutviksavsani) [reply action.yml]: Thanks for pointing it out, Seems like I had mistaken it as a requirement. I tried by removing the constant and pipeline seems fine

**Review by ethitter — APPROVED** [review]
